src/main/resources/assets/garnished/lang/lzh.json

```diff
@@ -7,113 +7,113 @@
-  "item.garnished.cinder_macadamia_fire_resistance": "Macadamia",
-  "item.garnished.honeyed_macadamia": "Macadamia",
-  "item.garnished.chocolate_glazed_macadamia": "Macadamia",
+  "item.garnished.macadamia": "澳洲胡桃",
+  "item.garnished.ungarnished_macadamia": "澳洲胡桃",
+  "item.garnished.cinder_macadamia": "澳洲胡桃",
+  "item.garnished.melted_cinder_macadamia": "澳洲胡桃",
+  "item.garnished.sweetened_macadamia": "澳洲胡桃",
+  "block.garnished.macadamia_sapling": "澳洲胡桃",
+  "item.garnished.cinder_macadamia_fire_resistance": "澳洲胡桃",
+  "item.garnished.honeyed_macadamia": "澳洲胡桃",
+  "item.garnished.chocolate_glazed_macadamia": "澳洲胡桃",
 
-  "item.garnished.peanut": "Peanut",
-  "item.garnished.ungarnished_peanut": "Peanut",
-  "item.garnished.cinder_peanut": "Peanut",
-  "item.garnished.melted_cinder_peanut": "Peanut",
-  "item.garnished.sweetened_peanut": "Peanut",
-  "block.garnished.peanut_sapling": "Peanut",
-  "item.garnished.cinder_peanut_effect": "Peanut",
-  "item.garnished.honeyed_peanut": "Peanut",
-  "item.garnished.chocolate_glazed_peanut": "Peanut",
+  "item.garnished.peanut": "土豆",
+  "item.garnished.ungarnished_peanut": "土豆",
+  "item.garnished.cinder_peanut": "土豆",
+  "item.garnished.melted_cinder_peanut": "土豆",
+  "item.garnished.sweetened_peanut": "土豆",
+  "block.garnished.peanut_sapling": "土豆",
+  "item.garnished.cinder_peanut_effect": "土豆",
+  "item.garnished.honeyed_peanut": "土豆",
+  "item.garnished.chocolate_glazed_peanut": "土豆",
 
-  "item.garnished.hazelnut": "Hazelnut",
-  "item.garnished.ungarnished_hazelnut": "Hazelnut",
-  "item.garnished.cinder_hazelnut": "Hazelnut",
-  "item.garnished.melted_cinder_hazelnut": "Hazelnut",
-  "item.garnished.sweetened_hazelnut": "Hazelnut",
-  "block.garnished.hazelnut_sapling": "Hazelnut",
-  "item.garnished.cinder_hazelnut_speed_potent": "Hazelnut",
-  "item.garnished.honeyed_hazelnut": "Hazelnut",
-  "item.garnished.chocolate_glazed_hazelnut": "Hazelnut",
+  "item.garnished.hazelnut": "榛子",
+  "item.garnished.ungarnished_hazelnut": "榛子",
+  "item.garnished.cinder_hazelnut": "榛子",
+  "item.garnished.melted_cinder_hazelnut": "榛子",
+  "item.garnished.sweetened_hazelnut": "榛子",
+  "block.garnished.hazelnut_sapling": "榛子",
+  "item.garnished.cinder_hazelnut_speed_potent": "榛子",
+  "item.garnished.honeyed_hazelnut": "榛子",
+  "item.garnished.chocolate_glazed_hazelnut": "榛子",
 
-  "item.garnished.garnished_sweet_berries": "Sweet Berries",
-  "item.garnished.honeyed_sweet_berries": "Sweet Berries",
+  "item.garnished.garnished_sweet_berries": "甜莓",
+  "item.garnished.honeyed_sweet_berries": "甜莓",
 
-  "item.garnished.nut_mix": "Nut Mix",
-  "item.garnished.sweetened_nut_mix": "Nut Mix",
-  "item.garnished.honeyed_nut_mix": "Nut Mix",
-  "item.garnished.chocolate_glazed_nut_mix": "Nut Mix",
+  "item.garnished.nut_mix": "混紜堅果",
+  "item.garnished.sweetened_nut_mix": "混紜堅果",
+  "item.garnished.honeyed_nut_mix": "混紜堅果",
+  "item.garnished.chocolate_glazed_nut_mix": "混紜堅果",
```

Comment:
(Unrelated to specific commit of text) Some lines are missing from this file (refer to previous comments), please add the default english translations or placeholders for the time being to avoid any conflictions.